internal/component/registry.go

```diff
@@ -149,6 +149,9 @@ type Registration struct {
 	// Build should construct a new component from an initial Arguments and set
 	// of options.
 	Build func(opts Options, args Arguments) (Component, error)
+
+	// Community is true if the component is a community component.
+	Community bool
```

Comment:
silly nit: can we put this after Stability? It seems semantically related to stability.